modules/ppcp-wc-subscriptions/src/WcSubscriptionsModule.php

```diff
@@ -496,4 +527,41 @@ function ( array $supports ) use ( $c ): array {
 			}
 		);
 	}
+
+	/**
+	 * Adds the given WC payment token into the given WC Order.
+	 *
+	 * @param WC_Order $wc_order
+	 * @param int $wc_payment_token_id
+	 * @param string $return_url
+	 * @param SessionHandler $session_handler
+	 * @return array
```

Comment:
I think this is always `string[]`
```suggestion
	 * @return array{result: string, redirect: string, errorMessage?: string}
```